packages/devtools_extensions/lib/src/api.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+enum DevToolsExtensionEventType {
+  ping,
```

Comment:
Are ping and pong the same as request and response?
Request and response seem to be more traditional.

Reply:
Added documentation. Ping and Pong are to verify the initial connection. This is what is done for posting messages to perfetto as well.